apps/widget-e2e/src/describer/modelAssessment/whatIfCounterfactuals/describeWhatIfCreate.ts

```diff
@@ -37,7 +37,11 @@ export function describeWhatIfCreate(dataShape: IModelAssessmentData): void {
         });
       cy.get(Locators.WhatIfCreateCounterfactualSortButton).click();
     });
-
+    it("should contain the output label name", () => {
+      cy.get(Locators.WhatIfColumnHeaders).contains(
+        dataShape.whatIfCounterfactualsData?.outputColumnName || ""
```

Comment:
Why are we allowing `""`? Doesn't that mean that it may also not be there?

Reply:
Yeah, not sure why. I copied the pattern from the test case above. @vinuthakaranth any reason why "" may be expected?